lib/json2csv.js

```diff
@@ -3,6 +3,7 @@
  */
 var os = require('os');
 var async = require('async');
+var _ = require('lodash');
```

Comment:
Please do `var get = require('lodash.get');` instead. It's much slimmer and a quicker install.